test/unit/component/parser/json/BpmnJsonParser.callActivity.test.ts

```diff
@@ -71,74 +61,365 @@ describe('parse bpmn as json for callActivity', () => {
+              event: {
+                bpmnKind: ShapeBpmnElementKind.EVENT_BOUNDARY,
+                isInterrupting: true,
+                attachedToRef: 'task_id_0',
+                eventDefinitionParameter: { eventDefinitionKind: 'message', eventDefinitionOn: EventDefinitionOn.EVENT },
+              },
+              task: { id: 'task_id_0' },
+            },
+            {
+              id: 'process_2',
+              callActivity: {
+                id: `call_activity_id_0`,
+                calledElement: 'process_1',
+                isExpanded,
+              },
+            },
+          ],
+        });
+
+        const model = parseJsonAndExpectOnlyFlowNodes(json, 3);
+
+        expect(model.flowNodes[0].bpmnElement.parentId).toBe('call_activity_id_0');
+        expect(model.flowNodes[1].bpmnElement.parentId).toBe('call_activity_id_0');
+        verifyShape(model.flowNodes[2], {
+          shapeId: 'shape_call_activity_id_0',
+          parentId: undefined,
+          bpmnElementId: 'call_activity_id_0',
+          bpmnElementName: undefined,
+          bpmnElementKind: ShapeBpmnElementKind.CALL_ACTIVITY,
+          bpmnElementCallActivityKind: ShapeBpmnCallActivityKind.CALLING_PROCESS,
+          bpmnElementMarkers: expectedBpmnElementMarkers,
+          bounds: { x: 346, y: 856, width: 45, height: 56 },
+        });
+      });
+
+      it(`should convert as Shape, when a ${expandedKind} call activity calling a process with sequence flow`, () => {
+        const json = buildDefinitions({
+          process: [
+            { id: 'process_1', sequenceFlow: { id: 'flow_2_id', sourceRef: 'event_2_id', targetRef: 'task_2_id' } },
+            {
+              id: 'process_2',
+              callActivity: {
+                id: `call_activity_id_0`,
+                calledElement: 'process_1',
+                isExpanded,
+              },
+            },
+          ],
+        });
+
+        const model = parseJsonAndExpectOnlyEdgesAndFlowNodes(json, 1, 1);
+
+        verifyShape(model.flowNodes[0], {
+          shapeId: 'shape_call_activity_id_0',
+          parentId: undefined,
+          bpmnElementId: 'call_activity_id_0',
+          bpmnElementName: undefined,
+          bpmnElementKind: ShapeBpmnElementKind.CALL_ACTIVITY,
+          bpmnElementCallActivityKind: ShapeBpmnCallActivityKind.CALLING_PROCESS,
+          bpmnElementMarkers: expectedBpmnElementMarkers,
+          bounds: { x: 346, y: 856, width: 45, height: 56 },
+        });
+
+        verifyEdge(model.edges[0], {
+          edgeId: 'edge_flow_2_id',
+          bpmnElementId: 'flow_2_id',
+          bpmnElementSourceRefId: 'event_2_id',
+          bpmnElementTargetRefId: 'task_2_id',
+          waypoints: [
+            { x: 45, y: 78 },
+            { x: 51, y: 78 },
+          ],
+        });
+      });
+
+      it(`should convert as Shape, when a ${expandedKind} call activity (calling process) (with/without name) is an attribute of 'process'`, () => {
```

Comment:
can we simplify the 2 parenthesis groups? In make the title hard to understand.
We are in a describe about 'call activity calling process' so I guess we can drop the reference to 'calling process'